File: test/fixtures/output/java/okhttp/text-plain.java
OkHttpClient client = new OkHttpClient();

MediaType mediaType = MediaType.parse("text/plain");
RequestBody body = RequestBody.create(mediaType, "Hello World");
Request request = new Request.Builder()
  .url("https://httpbin.org/anything")
  .post(body)
  .addHeader("content-type", "text/plain")
  .build();

Response response = client.newCall(request).execute();

File: src/targets/kotlin/okhttp/fixtures/text-plain.kt
val client = OkHttpClient()

val mediaType = MediaType.parse("text/plain")
val body = RequestBody.create(mediaType, "Hello World")
val request = Request.Builder()
  .url("https://httpbin.org/anything")
  .post(body)
  .addHeader("content-type", "text/plain")
  .build()

val response = client.newCall(request).execute()